cli/src/etos_client/test_results/test_results.py

```diff
@@ -57,22 +57,14 @@ def __test_result(self, test_suites: list[TestSuite]) -> tuple[bool, str]:
         """Build test results based on events retrieved."""
         if not self.__has_failed(test_suites):
             return True, "Test suite finished successfully."
-
-        failures = 0
-        sub_suites = 0
-        for test_suite in test_suites:
-            failures += self.__count_sub_suite_failures(test_suite.sub_suites)
-            sub_suites += len(test_suite.sub_suites)
         messages = self.__fail_messages(test_suites)
         if len(messages) == 1:
             return False, messages[0]
         if messages:
             for message in messages[:-1]:
                 self.logger.error(message)
             return False, messages[-1]
-        if sub_suites == 0:
-            return False, "ETOS failed to start any test suites"
-        return False, f"{failures}/{sub_suites} test suites failed."
+        return False, f"Test case failures during test suite execution"
```

Comment:
Just to double-check, ignore if it is out-of-scope:

If the suite JSON file contains an empty list, will this be handled by this method? I remember I saw weird error messages in that case before.

Reply:
I added a check for empty suite definition list, and I also checked some other sad path cases with broken suite definitions. It seems to work sas expected.